.github/workflows/update.yml

```diff
@@ -0,0 +1,60 @@
+name: Update README and LaTeX Build File
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-files:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+      
+      - name: Update files with Python
+        run: |
+          python -c "\
+          import os, re
+
+          # Extract repository name
+          repo_name = os.getenv('GITHUB_REPOSITORY').split('/')[-1]
+
+          # Update README.md
+          with open('README.md', 'r+') as file:
```

Comment:
Prefer Python's pathlib. The steps should be more of

1. Read file into string
1. Modify the string based on regex rather than looping line by line over it
1. Write the full string back to the file

Same approach can be applied for the cmake list